.github/workflows/CICD.yml

```diff
@@ -595,17 +595,6 @@ jobs:
           *-pc-windows-msvc) STRIP="" ;;
         esac;
         outputs STRIP
-    - name: Install/setup prerequisites
```

Comment:
why did you remove this?

Reply:
It's duplicated a few lines below, where I modify it with the additional lines. I believe 39a6e6c75b53222306b747de7d99fd257573b03b was supposed to move it but instead copied it. If I don't remove it, I'd need to choose which to modify, or modify both, in any case becoming a likely spot for bit rot.

Reply:
Yes, mistakenly left in... deletion should be fine.
Modify the second copy.